storagenode/peer.go

```diff
@@ -259,5 +249,8 @@ func (peer *Peer) ID() storj.NodeID { return peer.Identity.ID }
 // Local returns the peer local node info.
 func (peer *Peer) Local() pb.Node { return peer.Kademlia.RoutingTable.Local() }
 
-// Addr returns the public address.
-func (peer *Peer) Addr() string { return peer.Public.Server.Addr().String() }
+// PublicAddr returns the public address.
+func (peer *Peer) PublicAddr() string { return peer.Server.PublicAddr().String() }
```

Comment:
Your life might be easier if you don't change this method name just yet. A second PR renaming frequent methods can make PR review faster.